### ISSUE TITLE
unknown option: tmate-server-ed25519-fingerprint

### ISSUE THREAD

**Issue by natema** [tmate-io/tmate-ssh-server#80]
I've compiled from source `tmate-ssh-server` on a Debian 9 server following the [official instructions](https://tmate.io/#host). 
I installed `tmate` from the official repository on a Ubuntu 18.04.4 LTS client. 
According to instructions displayed during the installation process on the server (if I recall correctly, it was the output of `./create_keys.sh`), I created a `.tmate.conf` file on the client and added the lines 
```
set -g tmate-server-host "🤓🤓🤓🤓🤓🤓"
set -g tmate-server-port 2222
set -g tmate-server-rsa-fingerprint "SHA256:e2gIWjLJJhGo+ybEahwdj0XDJ8xSxsXVPFWayzIejPg"
set -g tmate-server-ed25519-fingerprint "SHA256:UXysTnprv4zFPSvYvC5VS2HwLQF57pQSWkg8Mf48WP4"
```
I then started the tmate server with `sudo ./tmate-ssh-server -p 2222 -v` getting the output 
```
Loading key keys/ssh_host_rsa_key
[ssh_pki_import_privkey_base64] ssh_pki_import_privkey_base64: Trying to decode privkey passphrase=false
[ssh_pki_openssh_import] ssh_pki_openssh_import: Opening OpenSSH private key: ciphername: none, kdf: none, nkeys: 1\n
Loading key keys/ssh_host_ed25519_key
[ssh_pki_import_privkey_base64] ssh_pki_import_privkey_base64: Trying to decode privkey passphrase=false
[ssh_pki_openssh_import] ssh_pki_openssh_import: Opening OpenSSH private key: ciphername: none, kdf: none, nkeys: 1\n
Accepting connections on :2222
```
but when I run `tmate` on the client I get `/home/natema/.tmate.conf:8: unknown option: tmate-server-ed25519-fingerprint` and on the server it repeatedly print the message 
```
Connection accepted ip=91.163.122.21
Bootstrapping ssh client ip=91.163.122.21
Exchanging DH keys
[ssh_server_connection_callback] ssh_server_connection_callback: SSH client banner: SSH-2.0-libssh_0.7.0
[ssh_analyze_banner] ssh_analyze_banner: Analyzing banner: SSH-2.0-libssh_0.7.0
[ssh_socket_exception_callback] ssh_socket_exception_callback: Socket exception callback: 1 (0)
[ssh_socket_exception_callback] ssh_socket_exception_callback: Socket error: disconnected
Error doing the key exchange: Socket error: disconnected
```

**Comment by nicopace** [comment]
have you had any luck with this? i am having a similar issue, with the same `unknown option: tmate-server-ed25519-fingerprint` issue

**Comment by natema** [comment]
@nicopace no, shortly after posting the issue here I switched back to using the default tmate.io server :disappointed:

**Comment by nicopace** [comment]
I found my issue... I was using a different version of tmate in the server and the client... when I matched their versions, it worked out of the box... 
I used the docker-compose version here: https://github.com/kkoralsky/tmate-docker-compose

**Comment by natema** [comment]
Thanks for pointing that out, I also see that my server and client version don't match. I'll confirm that matching versions work as soon as I can test that.

**Comment by pbsds** [comment]
I've also run having this issue. I host with the `prod` docker image.
The tmate client on debian (2.2.1) has this issue, so i removed the ecdsa fingerprint from `.tmate.conf`, but then i get "Cannot authenticate server". I can reproduce this on Manjaro as well which has the most recent client (2.4.0). RSA fingerprints alone seem to not be sufficient.
